test/integration/create_article_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
+
+class CreateArticleTest < ActionDispatch::IntegrationTest
+  setup do
+    @test_user = User.create(username: "Victoria", email: "victoria@email.com",
+                              password: "password", admin: false)
+    sign_in_as(@test_user)
+  end
+
+  test "get new article form and create new article" do 
+    get "/articles/new"  
+    assert_response :success
+    assert_difference 'Article.count', 1 do
+      post articles_path, params: {article: { title: "Nice title", description: "Nice description", 
+                                  category_ids: [] } } # fails if I put categories inside - why?
```

Comment:
Wenn ich in das test "get new article form..."  mit `category_ids: [1,2]` laufen lassen, ich stoße auf einen Fehler. Und zwar:
```
ActiveRecord::RecordNotFound: Couldn't find all Categories with 'id': (1, 2) (found 0 results, but was looking for 2). Couldn't find Categories with ids 1, 2.
```
Ich nehme an, dass man ein Category irgendwie in `setup` erstellen kann. Das kann ich zwar machen mit `Category.create(name: "Travel")` (z.B.) aber ich weiß nicht, wie ich eine ID anhängen kann.

Wenn ich es leer lasse, besteht es das Test.

Wenn ihr wollt, kann ich das bearbeiten aber dann brauche ich eure Hilfe :)

Reply:
> Ich habe gerade bemerkt, dass das Category controller kein destroy action hat. Denkt ihr, dass dies eine gutes Feature wäre, die man hinzufügen könnte? Wenn ja, was wären die Auswirkungen, wenn ein Admin eine Kategorie löscht? Wie würden die Artikel, die dieser Kategorie zugeordnet sind, betroffen sein?
> 
> UPDATE: ich glaube, ich lass das erstmal. Es ist eher ein "Nice to have", das zu sein eigene PR haben könnte.

:+1: Wir können das in die Liste der möglichen Zusatz-Aufgaben übernehmen.

Reply:
Na, einfach `Category.create(name: "Travel", id: 1)`)

Reply:
Nice. Easy fix <3